app/components/work_history_item_component.rb

```diff
@@ -38,12 +38,24 @@ def organisation
   attr_accessor :item
 
   def formatted_start_date
+    if FeatureFlag.active?(:restructured_work_history)
```

Comment:
Is it right to feature flag this? We'll have to handle both kinds of work history (old and new structures) in production. I think `start_date_unknown` will only be set for "restructured" entries.

Reply:
My understanding was that the candidate side changes (the approximate flag) was behind a feature flag. I assumed therefore that things that depend on this flag should be too. However it looks like the DB changes have already been made so I can remove the flag.